index.js

```diff
@@ -28,4 +29,159 @@ linkAbout.addEventListener('click', () => {
+      tech3: 'JavaScript',
+      tech4: 'html',
+    },
+    linkVersion: 'https://ameerhamzakhalid.github.io/portfolio-ameer-hamza/',
+    linkSource: 'https://github.com/ameerhamzakhalid/',
+  },
+  {
+    id: 5,
+    url: './img/pop-img.png',
+    title: 'Keeping track of hundreds of components 5',
+    description: `Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the
+  industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type
+  and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and
+  typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
+  when an unknown printer took a galley of type and scrambled it 1960s with the relea`,
+    tecnologies: {
+      tech1: 'Rubi on rails',
+      tech2: 'css',
+      tech3: 'JavaScript',
+      tech4: 'html',
+    },
+    linkVersion: 'https://ameerhamzakhalid.github.io/portfolio-ameer-hamza/',
+    linkSource: 'https://github.com/ameerhamzakhalid/',
+  },
+];
+
+const cardWorkDetailsMobile = (index) => `
+  <article class="card-work-details-mobile">
+          <button type="button" class="btn-close-details">
+          <img src="./img/live-ic.svg" alt="see live icon">
+          </button>
+          <div class="card-image-details-mobile">
+              <img src=${dataCards[index].url} alt="Medical illustration sets">
+          </div>
+          <h2 class="card-title-mobile">
+              ${dataCards[index].title}
+          </h2>
+          <ul class="card-tags-mobile">
+              <li class="tag-mobile">${dataCards[index].tecnologies.tech1}</li>
+              <li class="tag-mobile">${dataCards[index].tecnologies.tech2}</li>
+              <li class="tag-mobile">${dataCards[index].tecnologies.tech3}</li>
+              <li class="tag-mobile">${dataCards[index].tecnologies.tech4}</li>
+          </ul>
+          <p class="card-paragraph-mobile">
+            ${dataCards[index].description}
+          </p>
+          <div class="card-button-mobile">
+                  <a href=${dataCards[index].linkVersion} class="btn-card-details-mobile">
+                  See Live
+                  <img src="./img/live-ic.svg" alt="see live icon">
+                  </a>
+                  <a href=${dataCards[index].linkSource} class="btn-card-details-mobile">
+                  See Sources
+                  <img src="./img/gith-white.svg" alt="see github icon">
+                  </a>
+          </div>
+          <script src="index.js"></script>
+  </article>
+  `;
+
+for (let i = 0; i < arrayBtnPopup.length; i += 1) {
+  arrayBtnPopup[i].addEventListener('click', () => {
+    document.body.innerHTML = cardWorkDetailsMobile(i);
+    const btnCloseDetailsMobile = document.querySelector('.btn-close-details');
+    btnCloseDetailsMobile.addEventListener('click', async () => {
+      window.location.reload();
+    });
+  });
+}
```

Comment:
- [ ] Good job Implementing this code to close the popup window but using the async syntax here is useless because the async syntax is made to deal with promises since JavaScript is a single threaded language we need to implement parallel work in the API that why we use aync syntax in some places but here it is useless so I think it would be better to remove it